src/svc/invocable-scripts/paths/scripts_scriptID_invoke.yml

```diff
@@ -18,11 +44,85 @@ post:
           $ref: '../schemas/ScriptInvocationParams.yml'
   responses:
     '200':
-      description: The result of the script execution.
+      description: |
+        Success.
+        The result of the script execution.
       content:
-        application/json:
+        text/csv:
           schema:
             $ref: '../schemas/ScriptHTTPResponseData.yml'
+          examples:
+            $ref: '../schemas/ScriptInvokeResponse.yml'
+    '400':
+      description: |
+        Bad request.
+      headers:
+        X-Platform-Error-Code:
+          description: |
+            The reason for the error.
+          schema:
+            type: string
+            example: "invalid"
+      content:
+        application/json:
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            invalidParameters:
+              summary: Invalid parameters have been passed into the script.
+              value: {
+                "code": "invalid",
+                "message": "invalid parameters provided"
+              }
+    "401":
```

Comment:
@jstirnaman 
When I `run make generate-all` i get this reference error
`Error opening file "/openapi/src/svc/invocable-scripts/schemas/AuthorizationError.yml" 
ENOENT: no such file or directory, open '/openapi/src/svc/invocable-scripts/schemas/Error.yml'
`

I added the file into the svc schemas folder to resolve the issue but I dont think thats a good solution. Additionally, I get a ref error in the docs-v2 when running the bundling command.

`[4] cloud/ref.yml:11146:11 at #/paths/~1api~1v2~1scripts~1{scriptID}~1invoke/post/responses/401

Can't resolve $ref

11144 |         type: string
11145 | "401":
11146 |   $ref: '#/paths/~1scripts/get/responses/401'
11147 | "404":
11148 |   content:
`

Would you happen to know whats causing this or what I might be doing incorrectly?